artiq/coredevice/fastino.py

```diff
@@ -128,7 +129,7 @@ def voltage_group_to_mu(self, voltage, data):
         for i in range(len(voltage)):
             v = self.voltage_to_mu(voltage[i])
             if i & 1:
-                v = data[i // 2] | (v << 16)
+                v = data[i // 2] | (v << int32(16))
             data[i // 2] = v
```

Comment:
```suggestion
                v = data[i // 2] | (v << 16)
            data[i // 2] = int32(v)
```

Would make the intention significantly clearer to me.

Reply:
Ack